tests/README.md

```diff
@@ -0,0 +1,23 @@
+# Pebble Integration Tests
+
+## Run Tests
+
+```bash
+go test -count=1 -tags=integration ./tests/
```

Comment:
Yeah, I have taken note of this in a previous comment.

Split or in `tests.yml`: I'm OK with both, but I lean slightly towards splitting because logically, it's a little different from unit tests (it requires building a binary); and functionally, we can utilize multiple runners to run it in parallel. It also makes debugging easier because we don't have to go through a whole lot unit test successful logs to get to the failed integration ones.

Reply:
Sounds good, let's split it.